imgee/storage.py

```diff
@@ -275,8 +282,12 @@ def clean_local_cache(expiry=24):
 def delete(stored_file, commit=True):
     """
     Delete all the thumbnails and images associated with a file, from local cache and S3.
-    Wait for the upload/resize to complete if queued for the same image.
     """
+    registry = imgee.registry
+
+    # remove from registry
+    registry.remove_keys_starting_with(stored_file.name)
```

Comment:
Yes, UUID. It removes all the keys matching `imgee:registry:<name>*`. So if there are thumbnails of the image being deleted in the registry, they are removed too.